### ISSUE TITLE
Examination of AVM firmware download area

### ISSUE THREAD

**Issue by FendVario1** [amosproj/amos2022ss01-firmware-downloader#24]
## User story
1. As a Developer
2. I want / need an overview of the AVM firmware download area.
3. So that it is easily possible to get an overview of how to provide a crawler for this page.

## Acceptance criteria
* Detailed list of every action necessary to download a firmware file
* If a login is required create a random e-Mail address and supply the pages login information within the created document
* What functionality the crawler has to provide (FTP; user login; extract URLs; ...)
* All of the above in a document in this repository

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
